### PR TITLE
Refactor continuous deployment strategy

### PR DESCRIPTION
**Pull request by guillett**
* Move from TravisCI to CircleCI

### CODE REVIEW

**Comment by guillett** [comment]
@MattiSG aurais-tu le temps de faire la revue de cette PR ? Merci.

File: circle.yml [hunk 1]
```diff
@@ -54,3 +54,10 @@ test:
   post:
     - killall --wait sc:  # wait until Sauce Connect closes the tunnel
         parallel: true
+
+deployment:
+  metal:
+    branch: master
+    commands:
+      # CircleCI rely on a private key that triggers /opt/mes-aides/update.sh deploy
```
Comment (MattiSG) [inline circle.yml]: > relies

File: circle.yml [hunk 2]
```diff
@@ -54,3 +54,10 @@ test:
   post:
     - killall --wait sc:  # wait until Sauce Connect closes the tunnel
         parallel: true
+
+deployment:
+  metal:
```
Comment (MattiSG) [inline circle.yml]: > A mention of why this is called “metal” and how deployment works in the README would be welcome.

File: .travis.yml [hunk 3]
```diff
@@ -1,15 +1,3 @@
 language: node_js
 node_js:
   - "6"
```
Comment (MattiSG) [inline .travis.yml]: Why aren't we dropping Travis altogether? Circle is faster, yet we'll keep waiting on Travis builds to finish.

Comment (guillett) [inline .travis.yml]: Fair point.

**Comment by guillett** [comment]
Build on CircleCI https://circleci.com/gh/sgmap/mes-aides-ui/1713

**Pull request merged**